docs/docsite/rst/dev_guide/style_guide/preferred_terms.rst

```diff
@@ -0,0 +1,56 @@
+.. _preferred_terms:
+
+Preferred terminology
+=====================
+
+.. contents::
+  :local:
+
+The following is the list of preferred terms for Ansible ecosystem projects.
+
+Ansible
+   Use on all occurences or `ansible` if reffering to the CLI command. Preference is to use this to refer to the Ansible community package.
```

Comment:
RST typically requires double backticks for highlighting.

Also: s/reffering/referring/